general/mm.cpp

```diff
@@ -107,7 +107,7 @@ void* mm::Insert(void *ptr, const size_t bytes)
 void *mm::Erase(void *ptr)
 {
    if (!config::usingMM()) { return ptr; }
-   if (config::gpuDisabled()) { return ptr; }
+   if (config::gpuDisabled()) { return ptr; } //I think by default the condition should be true.
```

Comment:
@camierjs , I suspect this is were the problem arises. By default, I think we want ``gpuDisabled()`` to return true. With the existing framework it was returning false. Side note -I added the comment so I can reference that line of code, I wasn't sure how else to do it.